content/en/docs/guides/managing-jx/common-tasks/create-custom-builder.md

```diff
@@ -11,7 +11,7 @@ aliases:
   - /getting-started/create-custom-builder
 ---
 
-In Jenkins X, it is possible to create your custom Builders (aka [POD templates](https://github.com/jenkinsci/kubernetes-plugin)) or overwrite existing onces. You just need to base your Docker
+In Jenkins X, (when using jenkins classic and not tekton...) it is possible to create your custom Builders (aka [POD templates](https://github.com/jenkinsci/kubernetes-plugin)) or overwrite existing onces. You just need to base your Docker
```

Comment:
Maybe instead of adding this note, the whole page should move under the "Deprecated documentation" section to make clear that it does not apply to Jenkins X + Boot? WDYT?

Reply:
I m good with both